examples/with-glamorous/pages/index.js

```diff
@@ -1,7 +1,14 @@
 import React from 'react'
-import { css } from 'glamor'
+import { style, rehydrate, css } from 'glamor'
 import glamorous from 'glamorous'
 
+// Adds server generated styles to glamor cache.
```

Comment:
We're rehydrating the CSS by shoving the computed styles into the DOM in the `head`. This is actually being done by `glamor`. See here for more: https://github.com/threepointone/glamor/blob/master/docs/server.md

The `ids` are the hashes that are part of the generated classes uses in the CSS.

Here's an example I just pulled from a different project:
```js
__NEXT_DATA__ = {"props":{},"pathname":"/","query":{},"buildId":"-","buildStats":null,"assetPrefix":"","err":null,"ids":["14e33nr","151tlpa","63oe3q","35577l","wfyxwh","858wo0","2y5bqi","1nvbmsg","17186zx","cuyszr","ehn2ke","6l3qzw","1jfbwxw","gzrewh","1jh79q9","kv69lr","ubp1jq"]}
```